etl/steps/data/garden/war/2023-09-21/mie.meta.yml

```diff
@@ -1,3 +1,55 @@
+definitions:
+  all:
```

Comment:
We could have something along these lines. But at the moment I am not seeing how this could improve the YAML here? 

I guess in this case I could have something like

```yaml
definitions:
  all:
    display:
      selectedEntityNames:
        - Africa
        - Americas
        - Asia and Oceania
        - Europe
        - Middle East
```

which would propagate this to all `display.selectedEntityNames` fields? So I guess in general we could apply this sort of propagation to any field in metadata.

Reply:
I guess
```
definitions:
  common:
```
could be a good choice (or `shared` instead of `common`).
I thought that it could be enough with just one level, e.g.
```
common:
  ...
```
But I think having two levels gives you the freedom to define common entities, as well as individual ones, e.g.:
```
definitions:
  common:
    unit: test
  some_specific_variables_description: ...
```